.github/workflows/tryrebase.yml

```diff
@@ -0,0 +1,34 @@
+name: Rebase PR
+
+on:
+  repository_dispatch:
+    types: [try-rebase]
+
```

Comment:
You can add pull_request as a trigger here and create a mock PR to rebase on line 34.

Reply:
I think the testing in https://github.com/clee2000/pytorch/pull/6 should cover most of this

Reply:
oh how did i not notice that--that looks very similar yes